src/DependencyInjection/SonataAdminExtension.php

```diff
@@ -150,12 +150,9 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('exporter.xml');
         }
 
-        // NEXT_MAJOR : remove this block
-        if (method_exists(Definition::class, 'setDeprecated')) {
```

Comment:
only the if or the whole block?

Reply:
Probably the whole block, can you create a PR?

Reply:
Here you go: https://github.com/sonata-project/SonataAdminBundle/pull/5020

Reply:
no we need to keep it!